assets/js/_main.js

```diff
@@ -261,8 +266,13 @@ $(document).ready(function() {
       "11": ["installing-boot9strap-(kartdlphax)", "finalizing-setup"],
       "12": ["seedminer-(twn)", "bannerbomb3-fredtool-(twn)", "finalizing-setup"],
       "13": ["installing-boot9strap-(ssloth-browser)", "finalizing-setup"],
-	    "14": ["multiple-options", "finalizing-setup"],
-      "15": ["seedminer-(twn)", "bannerbomb3-fredtool-(twn)", "finalizing-setup"]
+      "14": ["multiple-options", "finalizing-setup"],
+      "15": ["seedminer-(twn)", "bannerbomb3-fredtool-(twn)", "finalizing-setup"],
+      "16": ["homebrew-launcher-(super-skaterhax)", "dumping-movable-(nimhax)", "installing-boot9strap-(frogtool)", "finalizing-setup"],
+      "17": ["homebrew-launcher-(super-skaterhax)", "installing-boot9strap-(hbl-usm)", "finalizing-setup"],
+      "18": ["homebrew-launcher-(super-skaterhax)", "multiple-options", "finalizing-setup"],
+      "19": ["homebrew-launcher-(super-skaterhax)", "dumping-movable-(nimhax)", "multiple-options", "finalizing-setup"],
```

Comment:
wouldn't hbl (skater) -> nimhax -> bb3 -> fredtool also be a valid path? (That would be the only point of getting people through BB3, right? Since if they're not doing HBL-USM, their shoulder buttons must be broken?)

Reply:
selecting bannerbomb3 brings them to after seedminer, if i recall correctly, so it doesnt matter